frontend/src/components/student/questions/question-elements/QuestionText.tsx

```diff
@@ -1,13 +1,21 @@
 import React from "react";
 import { Text } from "@chakra-ui/react";
 
+import { getLetterFromNumber } from "../../../../utils/GeneralUtils";
+
 interface QuestionTextProps {
+  index: number;
   questionText: string;
 }
 const QuestionText = ({
+  index,
   questionText,
 }: QuestionTextProps): React.ReactElement => {
-  return <Text textStyle="subtitle1">{questionText}</Text>;
+  return (
+    <Text textStyle="subtitle1">
+      {`${getLetterFromNumber(index)}. ${questionText}`}
+    </Text>
```

Comment:
we should only show a letter in the case that there is > 1 sub-question. does this handle that logic? can we also refactor the question editor to use the common function?

Reply:
<img width="927" alt="image" src="https://github.com/uwblueprint/jump-math/assets/69697180/ef8c97f4-a511-4d48-843d-81d033c1a1f0">